### PR TITLE
As dotnet tool

### PR DESCRIPTION
**Pull request by kopfrechner**
with options to generate an example config and use a custom confg to generate a report from toggl

### CODE REVIEW

File: Adliance.Togglr/Adliance.Togglr.csproj [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <Nullable>enable</Nullable>
+    <LangVersion>Latest</LangVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    
+    <AssemblyName>Adliance.Togglr</AssemblyName>
+    <RootNamespace>Adliance.Togglr</RootNamespace>
+    <AssemblyVersion>0.0.0.0</AssemblyVersion>
+    <Version>0.0.0</Version>
+    <FileVersion>0.0.0.0</FileVersion>
+    <Authors>Hannes Sachsenhofer</Authors>
```
Comment (saxx) [inline Adliance.Togglr/Adliance.Togglr.csproj:16]: I think "Christoph Kopf" is missing here now, too ;)

File: Adliance.Togglr/TogglrConfigurationGeneratorParameters.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using CommandLine;
+
+namespace Adliance.Togglr
+{
+    [Verb("configuration-generator", HelpText = "Generate a template 'configuration.json' in the current folder")]
```
Comment (saxx) [inline Adliance.Togglr/TogglrConfigurationGeneratorParameters.cs:5]: I think the verb should be an actual verb, like `generate-configuration`

**Review by saxx — CHANGES_REQUESTED** [review]

**Comment by saxx** [comment]
Thank you, great additions! Could you please also be so kind and update the readme to explain the new command line options?

**Comment by kopfrechner** [comment]
Added requested changes :)

**Review by saxx — APPROVED** [review]

**Pull request merged**